Makefile

```diff
@@ -80,17 +82,17 @@ test-php-phpstan: vendor-bin/phpstan/vendor
 
 .PHONY: test-acceptance-api
 test-acceptance-api: ## Run API acceptance tests
-test-acceptance-api: vendor-bin/behat/vendor
+test-acceptance-api: $(acceptance_test_deps)
 	BEHAT_BIN=$(BEHAT_BIN) ../../tests/acceptance/run.sh --remote --type api
 
 .PHONY: test-acceptance-cli
 test-acceptance-cli: ## Run CLI acceptance tests
-test-acceptance-cli: vendor-bin/behat/vendor
+test-acceptance-cli: $(acceptance_test_deps)
```

Comment:
Should not we be consistent with similar target's dependencies? Like, in phpunit, we do:
```
test-php-unit: vendor
```

In core, we do `$(` but in apps, we mix these.

Reply:
The Makefile is a bit different in every app. When I add something new, I try to be consistent in all Makefiles.

Some day someone could look through all the Makefiles and make the format exactly the same.